app.py

```diff
@@ -135,35 +135,63 @@ def callback_view():
     return redirect(session.pop('next'))
   return redirect(url_for('demo_view'))
 
-@app.route('/hook/<pull_request_id>/<path:object_key>')
-def hook_view(pull_request_id, object_key):
-  repo_name = request.args.get('repo_name')
+def hook(args):
```

Comment:
Since this is a general hook function now, it might make sense to move it into a file in `helpers`, keeping this file only for route definitions.